man/py_select_interpreter.Rd

```diff
@@ -0,0 +1,11 @@
+% Generated by roxygen2: do not edit by hand
```

Comment:
qq: how is this get generated? What command to run? or it's in the pre-commit hook?

Reply:
This works but probably should be in docs or automated: `R --no-save -e "devtools::document()"`

Reply:
Thank you! Probably it can be added to README for whoever is not familiar with this generation.